aspnetcore/web-api/advanced/conventions.md

```diff
@@ -3,40 +3,60 @@ title: Use web API conventions
 author: pranavkm
 description: Learn about web API conventions in ASP.NET Core.
 monikerRange: '>= aspnetcore-2.2'
-ms.author: pranavkm
+ms.author: scaddie
 ms.custom: mvc
-ms.date: 11/13/2018
+ms.date: 11/30/2018
 uid: web-api/advanced/conventions
 ---
 # Use web API conventions
 
-ASP.NET Core 2.2 introduces a way to extract common [API documentation](xref:tutorials/web-api-help-pages-using-swagger) and apply it to multiple actions, controllers, or all controllers within an assembly. Web API conventions are a substitute for decorating individual actions with [[ProducesResponseType]](xref:Microsoft.AspNetCore.Mvc.ProducesResponseTypeAttribute). It allows you to define the most common "conventional" return types and status codes that you return from your action with a way to select the convention method that applies to an action.
+By [Pranav Krishnamoorthy](https://github.com/pranavkm) and [Scott Addie](https://github.com/scottaddie)
 
-By default, ASP.NET Core MVC 2.2 ships with a set of default conventions, `Microsoft.AspNetCore.Mvc.DefaultApiConventions`. The conventions are based on the controller that ASP.NET Core scaffolds. If your actions follow the pattern that scaffolding produces, you should be successful using the default conventions.
+ASP.NET Core 2.2 introduces a way to extract common [API documentation](xref:tutorials/web-api-help-pages-using-swagger) and apply it to multiple actions, controllers, or all controllers within an assembly. Web API conventions are a substitute for decorating individual actions with [[ProducesResponseType]](xref:Microsoft.AspNetCore.Mvc.ProducesResponseTypeAttribute).
```

Comment:
Let's get this ready for 3.0 release
ASP.NET Core 2.2 and later includes a way to ...